src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -740,6 +760,26 @@ struct jit_uni_eltwise_generic : public jit_uni_eltwise_kernel, public jit_gener
     }
 
     inline void load_scalar(Xmm xmm_src, const Xbyak::Address &op, ov::element::Type src_prc, ov::element::Type dst_prc) {
+        if (src_prc == dst_prc) {
+            switch (src_prc.size()) {
+                case 4:
+                    uni_vmovss(xmm_src, op);
+                    break;
+                case 1:
+                    mov(reg_tmp_8, op);
+                    if (src_prc.is_signed()) {
```

Comment:
As long as you are not doing precision conversion here there is no need to distinguish between singned/unsigned types. Just copy 8bits from memory to xmm.

Reply:
simplified, thanks!